httpclient5/pom.xml

```diff
@@ -80,6 +80,7 @@
     <dependency>
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
+      <scope>test</scope>
```

Comment:
@j3graham Can't we drop `commons-codec` entirely? Do we really use it in any of the tests?

Reply:
Should be able to. Right now my test iterates over lots of inputs and compares what each codec returns. This is likely overkill - I will update the tests to use constant expected values for a reasonable number of points.

Reply:
It's removed